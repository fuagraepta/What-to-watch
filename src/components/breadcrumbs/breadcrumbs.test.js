import React from 'react';
import renderer from 'react-test-renderer';

import BreadCrumbs from './breadcrumbs.jsx';

describe(`The component is rendered correctly`, () => {
  it(`Breadcrumbs renders correctly`, () => {
    const breadcrumbs = renderer
    .create(
        <BreadCrumbs
          title = {`The Grand Budapest Hotel`}
        />
    );

    expect(breadcrumbs).toMatchSnapshot();
  });
});
