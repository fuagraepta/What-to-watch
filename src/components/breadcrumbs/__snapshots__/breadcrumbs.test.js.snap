// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The component is rendered correctly Breadcrumbs renders correctly 1`] = `
<nav
  className="breadcrumbs"
>
  <ul
    className="breadcrumbs__list"
  >
    <li
      className="breadcrumbs__item"
    >
      <a
        className="breadcrumbs__link"
        href="movie-page.html"
      >
        The Grand Budapest Hotel
      </a>
    </li>
    <li
      className="breadcrumbs__item"
    >
      <a
        className="breadcrumbs__link"
      >
        Add review
      </a>
    </li>
  </ul>
</nav>
`;
